service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -55,11 +55,29 @@ export NXF_MODE=google
 
 sudo apt-get update
 
+#######################################
+# Install required JDK and set it as default (debian)
+#######################################
+function install_java() {
+  curl -Os https://download.oracle.com/java/17/latest/jdk-17_linux-x64_bin.deb
+  sudo apt-get install -y ./jdk-17_linux-x64_bin.deb
+  sudo update-alternatives --install /usr/bin/java java /usr/lib/jvm/jdk-17/bin/java 1
+  sudo update-alternatives --set java /usr/lib/jvm/jdk-17/bin/java
+}
+
 if [[ -n "$(which java)" ]];
 then
-  echo "java is installed"
+  readonly CUR_JAVA_VERSION=$(java -version 2>&1 | grep 'version' 2>&1 | awk -F\" '{ split($2,a,"."); print a[1]}')
+  if [[ "${CUR_JAVA_VERSION}" -lt 17 ]];
+  then
+    echo "current java version is ${CUR_JAVA_VERSION}, installing java 17"
```

Comment:
nit can `current` and `java` be capitalized? Same with lines 76, 79

Reply:
Sure think, had kept the capitalization consistent earlier